grapher/barCharts/DiscreteBarChart.tsx

```diff
@@ -201,13 +201,13 @@ export class DiscreteBarChart
         ]
     }
 
-    @computed private get yAxis(): AxisConfig {
+    @computed private get yAxisConfig(): AxisConfig {
         return this.manager.yAxis || new AxisConfig()
     }
 
-    @computed private get axis(): HorizontalAxis {
+    @computed get yAxis(): HorizontalAxis {
```

Comment:
@MarcelGerber not urgent at all, curious to hear your thoughts!

How do you feel about these renames to make every chart conform to the new `xAxis` and `yAxis` in [`ChartInterface`](https://github.com/owid/owid-grapher/pull/960/files#diff-4e3c3cd7daf006f3c0a7a6fe5cc32c39ed7f976a24b51faff2a2e3cd2a7c1555R30-R31)?

```ts
interface ChartTransform {
...
    yAxis?: HorizontalAxis | VerticalAxis
    xAxis?: HorizontalAxis | VerticalAxis
...
}
```

It seems necessary in order to get faceting working right – it needs to collapse uniform axes that repeat, and also needs to derive and pass down the correct Grapher props depending on whether it's X or Y axis. So this leads to calling the horizontal axis `yAxis` on DiscreteBars.

There is already this inconsistency in the config, so I thought it might be reasonable to propagate it to the charts themselves.

Reply:
Yep that's fine with me, no objections.
It's maybe not ideal but if it makes life easier, let's do it.